corehq/apps/domain/views.py

```diff
@@ -2525,6 +2528,21 @@ def page_context(self):
 
 @login_and_domain_required
 @require_superuser
+@require_GET
+def feature_flag_diff(request, domain):
+    params = json_request(request.GET)
+    other_domain = params.get('domain')
+    diff = []
+    if Domain.get_by_name(other_domain):
+        diff = [(t.slug, t.label) for t in all_toggles()
+                if t.enabled(request.domain) and not t.enabled(other_domain)]
+        diff.sort(cmp=lambda x, y: cmp(x[1], y[1]))
+        diff = ["<a href='{}'>{}</a>".format(reverse(ToggleEditView.urlname, args=[d[0]]), d[1]) for d in diff]
+    return json_response(diff)
```

Comment:
Would be nice to do client-side, but toggles.js only knows about the current domain's flags, not the one you're copying to.